.gitignore

```diff
@@ -1,2 +1,4 @@
 target/
 **/*.rs.bk
+src/asset.in
+frontend/dist/
```

Comment:
I think updating the assets manually is fine, it's the "needing to copy/paste the new hashes into the source code" that's the real issue. 

Hm.